client/components/domains/registrant-extra-info/ca-form.jsx

```diff
@@ -35,6 +35,8 @@ export class RegistrantExtraInfoCaForm extends React.PureComponent {
 	static propTypes = {
 		contactDetails: PropTypes.object.isRequired,
 		ccTldDetails: PropTypes.object.isRequired,
+		onContactDetailsChange: PropTypes.func,
```

Comment:
Not really related to your PR but it might be worth filling in the missing `updateContactDetailsCache` (required) proptype here too.